.jsdoc.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
     copyright: 'Copyright 2020 Google LLC',
     includeDate: false,
     sourceFiles: false,
-    systemName: '@google-cloud/documentai',
+    systemName: 'documentai',
```

Comment:
I don't think we want this change - I wonder what's in `.repo-metadata.json`?  I think that's how this field gets populated.